apps/src/code-studio/pd/international_optin/InternationalOptin.jsx

```diff
@@ -0,0 +1,248 @@
+import React, {PropTypes} from 'react';
+import FormController from '../form_components/FormController';
+import FormComponent from '../form_components/FormComponent';
+import DatePicker from '../workshop_dashboard/components/date_picker';
+import moment from 'moment';
+import {DATE_FORMAT} from '../workshop_dashboard/workshopConstants';
+import {
+  Row,
+  Col,
+  ControlLabel,
+  FormGroup
+} from 'react-bootstrap';
+
+export default class InternationalOptin extends FormController {
+  static propTypes = {
+    accountEmail: PropTypes.string.isRequired,
+    labels: PropTypes.object.isRequired
+  };
+
+  /**
+   * @override
+   */
+  onSuccessfulSubmit(data) {
+    window.location = `/pd/international_optin/${data.id}/thanks`;
+  }
+
+  /**
+   * @override
+   */
+  serializeFormData() {
+    const formData = super.serializeFormData();
+    //Object.assign(formData['form_data'], this.getDistrictData());
+
+    return formData;
+  }
```

Comment:
What is the point of this override if it just calls super and returns the result?